### ISSUE TITLE
Make it easy to check progress of a commit propagating through the DAG

### ISSUE THREAD

**Issue by gabrielgrant** [pachyderm/pachyderm#3983]
**What is the goal / desired outcome?**

I want to get the progress of how my commit to an input repo is trickling down the DAG, including being able to easily do a binary check of whether or not it has finished trickling 

**If there is a way to accomplish this today via workaround, what does that require?**

1. Learn what subvenance is
2. get subvenance (which is not directly exposed through either the CLI or dash)
3. for each subvenant commit, check whether it is still open

**(Optional) What is your proposal for a feature to solve this?**

Add more options for flush-commit (or some similar endpoint with similar functionality):

`flush-commit --blocking=false` -- immediately return the list of downstream commits that are finished and exit. perhaps use exit code to indicate whether there are more commits to be completed?

Edit: according to @jdoliner  this is already the behavior of `flush-commit` ~`flush-commit --progress` -- stream back the list of downstream commits as they are finished (any that are already complete should stream back immediately)~

**Comment by jdoliner** [comment]
This is much better implemented as fields on `CommitInfo` like so:

```
message CommitInfo {
  ...
  int64 subvenance_success = 15;
  int64 subvenance_failure = 16;
}
```

Which keep a count of how many subvenant commits have been closed with output trees (success) and how many have been closed without one (failure). This could then be reflected `list commit` as a progress meter similar to those in `list job`.

**Comment by jdoliner** [comment]
Implemented in #4203
